modules/candidate_list/php/candidate_list.class.inc

```diff
@@ -218,7 +218,7 @@ class Candidate_List extends \NDB_Menu_Filter
         parent::_setupPage($name, $page, $identifier, $commentID, $formname);
         include_once __DIR__
             . "/../../candidate_parameters/php/"
-            . "NDB_Form_candidate_parameters.class.inc";
+            . "candidate_parameters.class.inc";
```

Comment:
this `include_once` should be replaced by `$cp_module = \Module::factory('candidate_parameters')` to autoload the namespace.